components/my-account/Lists/ViewLists/ViewLists.tsx

```diff
@@ -39,33 +53,102 @@ const ViewLists = (props: ListsProps) => {
     sortBy: publicRuntimeConfig.b2bList.sortBy,
     filter: publicRuntimeConfig.b2bList.filter,
   })
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [listData, setListData] = useState<CrWishlist>()
 
   // screen size declared
   const theme = useTheme()
   const mdScreen = useMediaQuery<boolean>(theme.breakpoints.up('md'))
   const { t } = useTranslation('common')
   // this will be used for creating new list
   const { user } = useAuthContext()
-  const userId = user?.userId
+  const userId = user?.id
```

Comment:
don't need to save these in a separate variable

Reply:
done